public/swagger/SwaggerPage.tsx

```diff
@@ -49,16 +49,34 @@ export const Page = () => {
     return urls;
   });
 
+  const [userView, setUserView] = useState<UserView>();
+
   const namespace = useAsync(async () => {
     const response = await fetch('api/frontend/settings');
     if (!response.ok) {
       console.warn('No settings found');
-      return '';
+      return 'default';
```

Comment:
@Proximyst -- i think this gives us the same as https://github.com/grafana/grafana/pull/96686